CHANGELOG.md

```diff
@@ -6,15 +6,22 @@ All user visible changes to `cucumber` crate will be documented in this file. Th
 
 
 
-## [0.15.2] · 2022-10-??
-[0.15.2]: /../../tree/v0.15.2
+## [0.16.0] · 2022-10-??
+[0.16.0]: /../../tree/v0.16.0
 
-[Diff](/../../compare/v0.15.1...v0.15.2) | [Milestone](/../../milestone/17)
+[Diff](/../../compare/v0.15.1...v0.16.0) | [Milestone](/../../milestone/17)
+
+### BC Breaks
+
+- Upgraded [`gherkin`] crate to 0.13 version. ([4cad49f])
```

Comment:
@ilslv we don't require `cucumber` users to use `gherkin` crate separately. If the user uses `cucumber` only, it's not a breaking change him.

Reply:
@tyranron the same logic could be applied to upgrading `clap` or even previous versions bump of `gherkin`. We've considered all of them as a breaking changes.

Reply:
@ilslv nope. Previous versions of `gherkin` had either API changes, or at that time `gherkin` was required to be imported separately (there was no re-export and examples pointed to do so).

Regarding the `clap` crate, the reasons are different:
1. Users do use `clap` in their code in conjunction with `cucumber`, not only import it from `cucumber` (we re-export too little). So a new major version of `clap` will break that code.
2. `clap` major version updates were really breaking for the APIs.

In this case, the new `gherkin` release is major only due to MSRV bump. No API changes there. And we expect users just to use `gherkin` types we expose in `cucumber`. I don't think that some custom `gherkin`-based machinery, which a user has and wants it to integrate with APIs exposed by `cucumber` is a real case here.